client/src/components/total_number_reviews/total_number_reviews.js

```diff
@@ -0,0 +1,7 @@
+const React = require('react');
+
```

Comment:
in your original file, you imported a css stylesheet! don't forget it if you need it :D

Reply:
i removed it cuz it's throwing an error on tests! will add it later!

Reply:
👍